app.js

```diff
@@ -1,22 +1,23 @@
+const labelElement = document.querySelector("#label");
+const imageElement = document.querySelector("#cake-img");
+const ingredientsListElement = document.querySelector(".ingredients");
+const recipeUrlElement = document.querySelector("#recipe-url");
```

Comment:
I just wonder, why you want to out this here, but nevermind, let's leave it like that=)) It's really a minor thing.

Reply:
@catalinac3 catalinac3 1 hour ago Author Collaborator
I think you are right that a variable should be defined near the code, but I still think this are global variables, because the function is re-making them everytime it is call, as global they would be made once, and be used different times and they would be organized for future purposes at the top of the field.

 
@catalinac3 catalinac3 24 minutes ago Author Collaborator
also because in a difference to other variables they are dom elements, if we were use them to anywhere else, we know where to find them, and we avoid remaking them all over.